crates/circuit/src/operations.rs

```diff
@@ -954,7 +959,41 @@ impl Operation for StandardGate {
                 )
             }),
             Self::CRXGate | Self::CRYGate | Self::CRZGate => todo!(),
-            Self::RGate => todo!(),
+            Self::RGate => Python::with_gil(|py| -> Option<CircuitData> {
+                let theta_expr = match &params[0] {
+                    Param::Float(theta) => Param::Float(*theta),
+                    Param::ParameterExpression(theta) => {
+                        Param::ParameterExpression(theta.clone_ref(py))
+                    }
+                    Param::Obj(_) => unreachable!(),
+                };
+                let (phi_expr1, phi_expr2) = match &params[0] {
```

Comment:
Isn't phi the second parameter:

```suggestion
                let (phi_expr1, phi_expr2) = match &params[1] {
```